### PR TITLE
fix cstyle error in src/replication_test.c #2003

### PR DESCRIPTION
**Pull request by divyanshkul**
Fixes [#2003](https://github.com/openebs/openebs/issues/2003)

### CODE REVIEW

**Comment by divyanshkul** [comment]
Ping @ashishranjan738 @vishnuitta @payes

**Review by vishnuitta — COMMENTED** [review]
There seems to be some new issues in cstyle check like #86, #329 lines of new file.
can you please run the './cstyle.pl <filename>' and provide the output?

**Comment by divyanshkul** [comment]
@vishnuitta  did not have any issues at #86, #329. Had some other ones, fixed them. Now all that is remaining are >80 characters in the code. I fixed most of them which included strings, like 
```
REPLICA_ERRLOG("Failed to send response"
" replica(%d)\n", ctrl_port);

```

but I don't know C so have no idea about fixing strings like
```
count = test_read_data(events[i].data.fd, (uint8_t *)data + recv_len, total_len - recv_len);

```

Rest all the cstyle issues are fixed.

**Comment by divyanshkul** [comment]
@vishnuitta Travis build is passing, why is there an error with Better Code Hub (it isn't showing me errors as well, only admins can see)

**Comment by divyanshkul** [comment]
Ping @vishnuitta

File: src/replication_test.c [hunk 1]
```diff
@@ -75,22 +75,24 @@ init_mdlist(char *vol_name)
 
 	mdlist_fd = open(mdpath, O_CREAT|O_RDWR, 0666);
 	if (mdlist_fd < 0) {
-		REPLICA_ERRLOG("Failed to open metadata file %s err(%d)\n",
+		REPLICA_ERRLOG("Failed to open"
```
Comment (vishnuitta) [inline src/replication_test.c]: this seems to be unnecessary change. This is not part of cstyle errors of this file replication_test.c

**Review by vishnuitta — CHANGES_REQUESTED** [review]
there are unnecessary changes. Fix only the errors shown in cstyle as per the issue#2003.

File: src/replication_test.c (hunk 1 above)
Comment (divyanshkul) [reply src/replication_test.c]: It was showing me error on this line (when I ran it on my pc) @vishnuitta

Comment (vishnuitta) [reply src/replication_test.c]: can I know the errors that you saw?

Comment (divyanshkul) [reply src/replication_test.c]: Cstyle character>80 warning

Comment (divyanshkul) [reply src/replication_test.c]: Cstyle character>80 warning @vishnuitta

Comment (vishnuitta) [reply src/replication_test.c]: but, there are not more than 75 characters in that line. Can that be the case due to tabwidth in your editor?
Please set tabwidth as 8 in your editor.

File: src/replication_test.c [hunk 2]
```diff
@@ -323,11 +327,11 @@ send_mgmt_ack(int fd, zvol_op_code_t opcode, void *buf, char *replica_ip,
 			/* adjust iovec length */
 			for (i = start; i < start + 1; i++) {
 				if (iovec[i].iov_len != 0 && iovec[i].iov_len > (size_t)rc) {
-					iovec[i].iov_base
-						= (void *) (((uintptr_t)iovec[i].iov_base) + rc);
-					iovec[i].iov_len -= rc;
-					break;
-				} else {
+						iovec[i].iov_base
```
Comment (vishnuitta) [inline src/replication_test.c]: here, indentation went wrong.

**Comment by divyanshkul** [comment]
@vishnuitta sorry, there was no cstyle error in that line (my mistake :') ). Rest fixed that line and fixed the indentation error. :)

**Comment by divyanshkul** [comment]
@vishnuitta please review now :)

**Comment by payes** [comment]
@divyanshkul cstyle check still seems to fail.
Can you please re-run cstyle on the file and check.

**Comment by divyanshkul** [comment]
@payes 
Now all that is remaining are >80 characters in the code. I fixed most of them which included strings, like 
```
REPLICA_ERRLOG("Failed to send response"
" replica(%d)\n", ctrl_port);

```

but I don't know C so have no idea about fixing strings like
```
count = test_read_data(events[i].data.fd, (uint8_t *)data + recv_len, total_len - recv_len);

```

Rest all the cstyle issues are fixed.
